models/reviews-model.js

```diff
@@ -6,13 +6,30 @@ exports.selectReviews = () => {
   });
 };
 
-exports.selectReviewsByID = (id) => {
+exports.selectReviewsByID = (review_id) => {
   return db
-    .query(`SELECT * FROM reviews WHERE review_id =$1`, [id])
+    .query(`SELECT * FROM reviews WHERE review_id =$1`, [review_id])
     .then(({ rows: [review] }) => {
       if (review) {
+       
         return review;
       }
       return Promise.reject({ status: 404, msg: "ID NOT FOUND" });
     });
 };
+
+exports.updateVotes = (review_id, votes) => {
+  if(!votes) {
```

Comment:
can do if(!votes.inc_votes). that will cover a body with missing key like {newVotes:1}